tests/test_advection.py

```diff
@@ -195,6 +195,45 @@ def test_advection_RK45(lon, lat, mode, rk45_tol):
     print(fieldset.RK45_tol)
 
 
+def test_conversion_3DCROCO():
+    """Test of the (SciPy) version of the conversion from depth to sigma in CROCO
+
+    Values below are retrieved using xroms and hardcoded in the method (to avoid dependency on xroms):
+    # x, y = 10, 20
+    # s_xroms = ds.s_w.values
+    # z_xroms = ds.z_w.isel(time=0).isel(eta_rho=y).isel(xi_rho=x).values
+    # lat, lon = ds.y_rho.values[y, x], ds.x_rho.values[y, x]
```

Comment:
```suggestion
    ```py
    x, y = 10, 20
    s_xroms = ds.s_w.values
    z_xroms = ds.z_w.isel(time=0).isel(eta_rho=y).isel(xi_rho=x).values
    lat, lon = ds.y_rho.values[y, x], ds.x_rho.values[y, x]
    ```
```

Reply:
Nice solution, didn't know this was possible in docstrings!